### ISSUE TITLE
numpy() is only available when eager execution is enabled.'

### ISSUE THREAD

**Issue by BFascione** [SciSharp/Keras.NET#114]
Python.Runtime.PythonException: 'NotImplementedError : numpy() is only available when eager execution is enabled

Can anyone please advise me on how to fix this as I'm new to Keras.net

FYI I'm just running the example that comes with Keras.net on the ImageExamples

it seems to run ok see below but then it throws up a exception Python.Runtime.PythonException: 'NotImplementedError : numpy() is only available when eager execution is enabled

can someone please tell me where I enable this in the example 

any help would be appreciated

regards
Brian

Using TensorFlow backend.
2020-05-03 14:24:26.918862: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_101.dll
Warning! tensorflowjs is not installed
x_train shape: (60000, 28, 28, 1)
60000 train samples
10000 test samples
WARNING:tensorflow:From C:\Users\brian\.conda\envs\Visual-Studio-01\lib\site-packages\tensorflow_core\python\ops\resource_variable_ops.py:1635: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.
Instructions for updating:
If using Keras pass *_constraint arguments to layers.
2020-05-03 14:24:29.729980: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library nvcuda.dll
2020-05-03 14:24:29.753567: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1555] Found device 0 with properties:
pciBusID: 0000:01:00.0 name: GeForce RTX 2080 Ti computeCapability: 7.5
coreClock: 1.635GHz coreCount: 68 deviceMemorySize: 11.00GiB deviceMemoryBandwidth: 573.69GiB/s
2020-05-03 14:24:29.758819: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_101.dll
2020-05-03 14:24:29.764055: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_10.dll
2020-05-03 14:24:29.769334: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cufft64_10.dll
2020-05-03 14:24:29.772830: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library curand64_10.dll
2020-05-03 14:24:29.778517: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusolver64_10.dll
2020-05-03 14:24:29.783312: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusparse64_10.dll
2020-05-03 14:24:29.791797: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2020-05-03 14:24:29.795109: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1697] Adding visible gpu devices: 0
Train on 60000 samples, validate on 10000 samples
2020-05-03 14:24:30.208301: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
2020-05-03 14:24:30.212515: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1555] Found device 0 with properties:
pciBusID: 0000:01:00.0 name: GeForce RTX 2080 Ti computeCapability: 7.5
coreClock: 1.635GHz coreCount: 68 deviceMemorySize: 11.00GiB deviceMemoryBandwidth: 573.69GiB/s
2020-05-03 14:24:30.218578: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_101.dll
2020-05-03 14:24:30.221553: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_10.dll
2020-05-03 14:24:30.224897: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cufft64_10.dll
2020-05-03 14:24:30.227853: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library curand64_10.dll
2020-05-03 14:24:30.230827: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusolver64_10.dll
2020-05-03 14:24:30.234250: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusparse64_10.dll
2020-05-03 14:24:30.237280: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2020-05-03 14:24:30.240540: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1697] Adding visible gpu devices: 0
2020-05-03 14:24:30.687474: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1096] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-05-03 14:24:30.690521: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1102]      0
2020-05-03 14:24:30.692436: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] 0:   N
2020-05-03 14:24:30.694827: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1241] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 8683 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2080 Ti, pci bus id: 0000:01:00.0, compute capability: 7.5)
Epoch 1/10
2020-05-03 14:24:31.189754: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_10.dll
2020-05-03 14:24:31.408182: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2020-05-03 14:24:32.344747: W tensorflow/stream_executor/gpu/redzone_allocator.cc:312] Internal: Invoking GPU asm compilation is supported on Cuda non-Windows platforms only
Relying on driver to perform ptx compilation. This message will be only logged once.
60000/60000 [==============================] - 8s 139us/step - loss: 0.3022 - accuracy: 0.9064 - val_loss: 0.0639 - val_accuracy: 0.9794
Epoch 2/10
60000/60000 [==============================] - 3s 52us/step - loss: 0.0957 - accuracy: 0.9713 - val_loss: 0.0439 - val_accuracy: 0.9849
Epoch 3/10
60000/60000 [==============================] - 4s 62us/step - loss: 0.0713 - accuracy: 0.9789 - val_loss: 0.0423 - val_accuracy: 0.9860
Epoch 4/10
60000/60000 [==============================] - 4s 73us/step - loss: 0.0588 - accuracy: 0.9827 - val_loss: 0.0354 - val_accuracy: 0.9881
Epoch 5/10
60000/60000 [==============================] - 3s 51us/step - loss: 0.0511 - accuracy: 0.9848 - val_loss: 0.0325 - val_accuracy: 0.9889
Epoch 6/10
60000/60000 [==============================] - 3s 52us/step - loss: 0.0444 - accuracy: 0.9861 - val_loss: 0.0288 - val_accuracy: 0.9902
Epoch 7/10
60000/60000 [==============================] - 3s 51us/step - loss: 0.0403 - accuracy: 0.9876 - val_loss: 0.0281 - val_accuracy: 0.9910
Epoch 8/10
60000/60000 [==============================] - 3s 51us/step - loss: 0.0363 - accuracy: 0.9883 - val_loss: 0.0270 - val_accuracy: 0.9901
Epoch 9/10
60000/60000 [==============================] - 3s 51us/step - loss: 0.0337 - accuracy: 0.9901 - val_loss: 0.0321 - val_accuracy: 0.9902
Epoch 10/10
60000/60000 [==============================] - 3s 51us/step - loss: 0.0302 - accuracy: 0.9906 - val_loss: 0.0275 - val_accuracy: 0.9905

**Comment by BFascione** [comment]
the error has now stopped coming up but it just seems to sit there at the end of epoch 10/10
Running PrimaIndiansDiabetics 
is this correct ?

**Comment by BFascione** [comment]
the  last comment is when running the example Basic Samples

I've no idea if that correct ? can anyone please tell me

**Comment by BFascione** [comment]
Just Realised need to remove the comment on what you need to run from the program.cs file
